src/actions/arcActions.ts

```diff
@@ -1064,7 +1065,7 @@ export function redeemProposal(daoAvatarAddress: string, proposal: IProposalStat
         eventWatcher.stopWatching()
       }
 
-      if (redemption.stakerBountyTokens) {
+      if (redemption.stakerBountyTokens && dao.genCount >= redemption.stakerBountyTokens) {
```

Comment:
`stakerBountyTokens` should already be `0` if greater than the dao's funds.
https://github.com/daostack/alchemy/blob/ea2160a9ca8aced422feea76ee24e4e758a32581/src/actions/arcActions.ts#L395

Reply:
oh, ok so that check isn't needed then

Reply:
oh but actually we don't want that line, because then we are showing the user a staking bounty of 0, when actually it should be more.

Reply:
Correct. Then we should remove that line.